src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -172,7 +173,14 @@ export function RenderNickname ({ agentId }) {
   const { loading, error, data } = useQuery(HolofuelCounterpartyQuery, {
     variables: { agentId }
   })
+
   if (loading) return <React.Fragment>Loading...</React.Fragment>
-  if (error) { return presentAgentId(agentId) }
-  return <React.Fragment>{data.holofuelCounterparty.nickname}</React.Fragment>
+  if (error) {
+    return <CopyAgentId agent={{ id: agentId, nickname: '' }}>
+      {presentAgentId(agentId)}
+    </CopyAgentId>
+  }
+  return <CopyAgentId agent={data.holofuelCounterparty}>
+    {data.holofuelCounterparty.nickname}
+  </CopyAgentId>
```

Comment:
I would do this a little differently.

```
if (data.holofuelCounterparty.nickname) {
  return <CopyAgentId agent={data.holofuelCounterparty}>
    {data.holofuelCounterparty.nickname}
  </CopyAgentId>
} else {
  return <CopyAgentId agent={{ id: agentId, nickname: '' }}>
      {presentAgentId(agentId)}
    </CopyAgentId>
}
```

This removes the loading state. I don't think we want loading states here, it will be distracting when it's running against the real, slow, dna. Or if we do show a loading state, it should be a very small icon, next to agent id which is otherwise always displayed, so the user has the experience that there is always data available. But that seems like something for UX team to figure out. For now I say we just remove the loading state.

(I realize this is was already the case in previous PRs, I'm only just noticing it here)

Reply:
I'm fine with removing loading from now, but definitely think we need to create mm question about this and continue to track this question. :+1: